avatar.html

```diff
@@ -0,0 +1,117 @@
+                <a href="#" class="navbar-list-item">documentation</a>
+            </li>
+
+            <li >
+                <a href="https://github.com/Ramandeep9898" class="navbar-list-item icons">
+                    <i class="fab fa-github"></i>
+                </a>
+            </li>
+
+            <li >
+                <a href="https://www.linkedin.com/in/ramandeep-kaur-0090a8192/" class="navbar-list-item icons"> 
+                    <i class="fab fa-linkedin"></i>
+                </a>
+            </li>
+
+            <li>
+                    <a href="#" class="navbar-list-item icons">
+                        <i class="fab fa-whatsapp"></i>
+                    </a>
+            </li>
+
+            <li>
+                <a href="https://twitter.com/Ramande099928" class="navbar-list-item icons">
+                    <i class="fab fa-twitter"></i>
+                </a>
+            </li>
+
+            <li>
+                <a href="#" class="navbar-list-item icons">
+                    <i class="fas fa-moon"></i>
+                </a>
+            </li>
+
+        </ol>
+    </nav>
+    <section class="doc">
+        <section class="component-navbar">
+            <h1 class="component-navbar-heading">Getting Started</h1>
+            <li class="component-navbar-item">
+                <a href="#" class="component-navbar-item-link">usage</a>
+            </li>
+            <hr class="hr-sidebar">
+            <h1 class="component-navbar-heading">Components</h1>
+            <ui>
+                <li class="component-navbar-item">
+                    <a href="#" class="component-navbar-item-link">Alert</a>
+                </li>
+                <li class="component-navbar-item">
+                    <a href="#" class="component-navbar-item-link">Alert</a>
+                </li>
+                <li class="component-navbar-item">
+                    <a href="#" class="component-navbar-item-link">Alert</a>
+                </li>
+                <li class="component-navbar-item">
+                    <a href="#" class="component-navbar-item-link">Alert</a>
+                </li>
+                <li class="component-navbar-item">
+                    <a href="#" class="component-navbar-item-link">Alert</a>
+                </li>
+                <li class="component-navbar-item">
+                    <a href="#" class="component-navbar-item-link">Alert</a>
+                </li>
+                <li class="component-navbar-item">
+                    <a href="#" class="component-navbar-item-link">Alert</a>
+                </li>
+                <li class="component-navbar-item">
+                    <a href="#" class="component-navbar-item-link">Alert</a>
+                </li>
+                
+                
+            
+            </ui>
+        </section>
+        <section class="component-doc">
+            <h1 class="component-doc-no">01</h1>
+            <h1 class="component-navbar-heading">avatar</h1>
```

Comment:
Dont use multiple H1 tags in same page. It is not a good practice. If you want to create a visual hirearchy of things use h1, h2, h3 and may be add different font size to each of them

Reply:
hey tahir , I have changed these this tags in my other branch which is branch of this branch .. so when I will merge that branch with this branch it will be visible thanku :) and I also added comments.